### DISCUSSION TITLE
Custom Plan for Transit.land API

### DISCUSSION THREAD

**Discussion by jayesh0405** [transitland/transitland#403] [Q&A]
Hey there @drewda and @irees,
we have tried to contact you guys about the custom plan using the interline email but haven't heard back!
can you look into it!

Thank you so much!

**Comment by drewda** [comment]
Hello, as we shared via email, we point potential customers first to the Transitland Professional plan that is listed at https://www.interline.io/transitland/plans-pricing/

For customers who require larger quantities or rates of API access, or whose organization's require custom terms, we are able to customize the Transitland Enterprise plan. In this case we ask potential customers to email info@interline.io with more information about their specific needs.

Thank you.
